utils/Cargo.toml

```diff
@@ -14,11 +14,11 @@ documentation = "https://docs.rs/commonware-utils"
 commonware-codec = { workspace = true }
 bytes = { workspace = true }
 futures = { workspace = true }
+rand = { workspace = true }
 thiserror = { workspace = true }
 
```

Comment:
To fix that wasm issue, you need to add that dependency line: https://github.com/commonwarexyz/monorepo/blob/40b0db0fc5aa7e725d03a6f510c6ea4f76f33328/runtime/Cargo.toml#L31-L34

Reply:
TODO: https://github.com/commonwarexyz/monorepo/issues/930